services/mock-blockchain/src/client.ts

```
@@ -0,0 +1,122 @@
+ */
+export class MockBlockchainClient {
+  private baseUrl: string;
+
+  constructor(baseUrl = 'http://localhost:3800') {
+    this.baseUrl = baseUrl;
+  }
+
+  private async request<T>(
+    endpoint: string, 
+    method: 'GET' | 'POST' | 'PUT' | 'DELETE' = 'GET',
+    body?: any
```

Biome (1.9.4)

[error] Unexpected any. Specify a different type.

any disables many type checking rules. Its use should be avoided.

(lint/suspicious/noExplicitAny)